Makefile

```diff
@@ -834,17 +834,19 @@ update-leaderboard:
 update-yearly-leaderboard:
 	hack/yearly-leaderboard.sh
 
-out/docker-machine-driver-kvm2: out/docker-machine-driver-kvm2-$(GOARCH)
+# after https://github.com/kubernetes/minikube/issues/19959 is fixed needs to become out/docker-machine-driver-kvm2: out/docker-machine-driver-kvm2-$(GOARCH)
+out/docker-machine-driver-kvm2: out/docker-machine-driver-kvm2--x86_64
 	$(if $(quiet),@echo "  CP       $@")
 	$(Q)cp $< $@
 
 out/docker-machine-driver-kvm2-x86_64: out/docker-machine-driver-kvm2-amd64
 	$(if $(quiet),@echo "  CP       $@")
 	$(Q)cp $< $@
 
-out/docker-machine-driver-kvm2-aarch64: out/docker-machine-driver-kvm2-arm64
```

Comment:
These commands can stay, but `out/docker-machine-driver-kvm2-arm64` needs to be removed from the `drivers` command.